.changes/nextrelease/clean-elses

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "type": "enhancement",
+        "category": "Aws",
```

Comment:
Since this applies to a lot of the namespaces, can you leave this as an empty string "" please?